_run/kube/Makefile

```
@@ -61,6 +116,8 @@ kube-deployments-rollout: #$(patsubst %, kube-deployment-rollout-%,$(KUSTOMIZE_I
 .PHONY: kube-setup-kube
 ifeq ($(GATEWAY_API),true)
 kube-setup-kube: kube-setup-hostname-operator-gateway
+else ifeq ($(CONFIDENTIAL_COMPUTE),true)
+kube-setup-kube: kube-setup-cc
```

checkmake (0.3.2)

[warning] Target "kube-setup-kube" defined multiple times (lines 118 and 120).

(uniquetargets)